routes/members.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express')
 const router = express.Router()
-const { getMembers, getIdleMembers, migrateUserRoles, deleteIsMember, moveToMembers } = require('../controllers/members')
+const { getMembers, getIdleMembers, migrateUserRoles, deleteIsMember, moveToMembers, archiveMembers } = require('../controllers/members')
```

Comment:
We use every function from controllers. Let's not destructure it. Naming it as `member` would be great, and then using it as `members.___`. You can see it in other routes files for reference.

Reply:
Yes, good point, I followed what others did, and if I changed it now we have to change a lot of code.

Reply:
we only have to make changes in the routes/member.js file. You can even use `find and replace` for speeding it up. And better to do changes now, in the future we might have to change 2x of code.

Reply:
We just have to make changes in routes/member.js. You can use `find and replace` if you want to speed it up. And better to do changes now, in the future we might have to change 2x of code.

Reply:
Yeah makes sense. Done. Thank you for pointing this out.

Reply:
@MehulKChaudhari do the same here the way you did for controllers/member, please. I think you forgot to do it here.

Reply:
Done. Thank you